common/xfer/plugin_spec.go

```diff
@@ -7,8 +7,8 @@ import (
 	"sort"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/mndrix/ps"
 	"github.com/ugorji/go/codec"
+	"github.com/weaveworks/ps"
```

Comment:
In this case you're lucky, as its a single package library that doesn't seem to reference itself.  Other more complicated libraries have references to themselves in their code, and cannot just be checked out in arbitrary paths.

We should be running on our own fork for a long time.  If the upstream author doesn't want to take the change, we should work with him to find a change that is suitable to both of us.  If this fails, we have to take on the maintenance of our fork (and for instance implement CI etc).